src/octree/mod.rs

```diff
@@ -99,8 +100,8 @@ pub fn to_meta_proto(octree_meta: &OctreeMeta, nodes: Vec<proto::OctreeNode>) ->
 // TODO(hrapp): something is funky here. "r" is smaller on screen than "r4" in many cases, though
 // that is impossible.
 fn project(m: &Matrix4<f64>, p: &Point3<f64>) -> Point3<f64> {
-    let q = m * Point3::to_homogeneous(*p);
-    Point3::from_homogeneous(q / q.w)
+    let q = m * p.to_homogeneous();
+    Point3::from_homogeneous(q).unwrap()
```

Comment:
we don't divide by `q.w` anymore?

Reply:
That's done by nalgebra, see https://docs.rs/nalgebra/0.20.0/nalgebra/geometry/struct.Point.html#method.from_homogeneous